evalml/tests/automl_tests/parallel_tests/test_automl_dask.py

```diff
@@ -205,10 +205,9 @@ def test_automl_immediate_quit(
         pipelines_per_batch=5,
         ensembling=False,
         text_in_ensembling=False,
-        pipeline_params={},
-        custom_hyperparameters=None,
+        search_parameters={},
     )
-    automl.automl_algorithm.allowed_pipelines = pipelines
+    automl.automl_algorithm._set_allowed_pipelines(pipelines)
```

Comment:
Is the tuner not created otherwise? Wondering why we're making this change

Reply:
Yep! With the changes that we made to iterative, just setting `allowed_pipelines` doesn't automatically create tuners for them.